docs/pages/versions/unversioned/sdk/encoding.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Encoding
+description: Standard TextEncoder and TextDecoder APIs available on all Expo-supported platforms.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo'
+packageName: 'expo'
+platforms: ['android', 'ios', 'web', 'tvos']
+---
+
+The standard URL API is available on all Expo-supported platforms.
+
+## Installation
+
+This API is part of the `expo` package. Refer to the [browser and server runtime support](https://caniuse.com/textencoder) for web and Node.js.
+
+## Usage
+
+`TextEncoder` and `TextDecoder` are available globally without needing to import them.
+
+```js
+// [104, 101, 108, 108, 111]
+const hello = new TextEncoder().encode('hello');
+
+// "hello"
+const text = new TextDecoder().decode(hello);
+```
+
+## TextEncoder
+
+The `TextEncoder` API is included in the Hermes engine. See the source code here: [facebook/hermes > TextEncoder.cpp](https://github.com/facebook/hermes/blob/9e2bbf8eda15936ee00aee4f8e024ceaa7cd800d/lib/VM/JSLib/TextEncoder.cpp#L1).
```

Comment:
```suggestion
The `TextEncoder` API is included in the Hermes engine. See the [source code in TextEncoder.cpp inside Hermes GitHub repository](https://github.com/facebook/hermes/blob/9e2bbf8eda15936ee00aee4f8e024ceaa7cd800d/lib/VM/JSLib/TextEncoder.cpp#L1).
```

Let's spell out where the source code is and what file it is.

Reply:
"inside the Hermes GitHub repository"

Reply:
Thanks, update the above suggestion.